ios/CameraView+Focus.swift

```diff
@@ -74,24 +74,64 @@ extension CameraView {
       }
 
       // in {0..1} system
-      let normalizedPoint = captureDevicePointConverted(fromLayerPoint: point)
+      var normalizedPoint: CGPoint
+      if let previewView = previewView as? PreviewView {
+        // previewView is of type PreviewView can use the built in captureDevicePointConverted
+        normalizedPoint = previewView.videoPreviewLayer.captureDevicePointConverted(fromLayerPoint: point)
+      } else {
+        normalizedPoint = captureDevicePointConverted(fromLayerPoint: point)
+      }
```

Comment:
Let's remove this separate check and just make sure the `captureDevicePointConverted` method works as expected. That way we are future proof - maybe there could be a different Preview View type in the future.